google/cloud/spanner/samples/postgresql_samples.cc

```diff
@@ -207,6 +215,444 @@ void DmlGettingStartedUpdate(google::cloud::spanner::Client client) {
+void CreateStoringIndex(
+    google::cloud::spanner_admin::DatabaseAdminClient client,
+    google::cloud::spanner::Database const& database) {
+  std::vector<std::string> statements = {
+      R"""(
+        CREATE INDEX AlbumsByAlbumTitle
+            ON Albums (AlbumTitle NULLS FIRST)
+            INCLUDE (MarketingBudget)
+      )""",
+  };
+  auto metadata =
+      client.UpdateDatabaseDdl(database.FullName(), statements).get();
+  if (!metadata) throw std::runtime_error(metadata.status().message());
+  std::cout << "Index added.\nNew DDL:\n" << metadata->DebugString();
+}
+// [END spanner_postgresql_create_storing_index]
+
+// [START spanner_postgresql_information_schema]
+void InformationSchema(
+    google::cloud::spanner_admin::DatabaseAdminClient admin_client,
+    google::cloud::spanner::Database const& database,
+    google::cloud::spanner::Client client) {
+  std::vector<std::string> statements = {
+      R"""(
+        CREATE TABLE Venues (
+            VenueId  BIGINT NOT NULL PRIMARY KEY,
+            Name     CHARACTER VARYING(1024) NOT NULL,
+            Revenues NUMERIC,
+            Picture  BYTEA
+        )
+      )""",
+  };
+  auto metadata =
+      admin_client.UpdateDatabaseDdl(database.FullName(), statements).get();
+  if (!metadata) throw std::runtime_error(metadata.status().message());
+  std::cout << "Table created.\nNew DDL:\n" << metadata->DebugString();
+
+  // Get all the user tables in the database. PostgreSQL uses the `public`
+  // schema for user tables. The table_catalog is equal to the database name.
+  // The `user_defined_` columns are only available for PostgreSQL databases.
+  auto sql = google::cloud::spanner::SqlStatement(R"""(
+      SELECT table_catalog, table_schema, table_name,
+             user_defined_type_catalog,
+             user_defined_type_schema,
+             user_defined_type_name
+          FROM INFORMATION_SCHEMA.tables
+          WHERE table_schema = 'public'
+  )""");
+  using RowType =
+      std::tuple<absl::optional<std::string>, std::string, std::string,
+                 absl::optional<std::string>, absl::optional<std::string>,
+                 absl::optional<std::string>>;
+  auto rows = client.ExecuteQuery(std::move(sql));
+  for (auto const& row : google::cloud::spanner::StreamOf<RowType>(rows)) {
+    if (!row) throw std::runtime_error(row.status().message());
+    std::string user_defined_type = "null";
+    if (std::get<3>(*row).has_value()) {
+      user_defined_type = std::get<3>(*row).value() + "." +
+                          std::get<4>(*row).value() + "." +
+                          std::get<5>(*row).value();
+    }
+    std::cout << "Table: " << std::get<2>(*row)
+              << " (User defined type: " << user_defined_type << ")\n";
+  }
+}
+// [END spanner_postgresql_information_schema]
+
+// [START spanner_postgresql_numeric_data_type]
+void NumericDataType(google::cloud::spanner::Client client) {
+  /*
+   * Remaining samples to add:
```

Comment:
Is this something we want to appear in the documentation / snippet?

Reply:
They're not grabbing any of this yet, and it will be filled out before they do.

But even if they were grabbing it, I think the answer would be "yes".

Reply:
That seems odd.  I do not think our own TODO lists should be part of the example code we display to customers.

Reply:
Fair enough.  I moved the comment outside the tags.